pyproject.toml

```diff
@@ -102,7 +102,7 @@ module = [
   "botocore.*",
   "bs4.*",
   "chromadb.*",
-  "clip.*",
+  "clickhouse_connect.*",
```

Comment:
We want the user to install it on their own when they need it. Don't need it here